salt/modules/acme.py

```diff
@@ -1,4 +1,4 @@
-# -*- coding: utf-8 -*-
```

Comment:
I think this line was changed accidentally. Would you mind changing this line back?